contrib/openshift/manifests/manifests.yaml

```diff
@@ -56,7 +56,7 @@ objects:
               command:
                 - sh
                 - '-c'
-                - rm -rf /tmp/sha* /tmp/fetch.* /tmp/rpmscanner.* && exec clair
+                - rm -rf /var/tmp/sha* /var/tmp/fetch.* /var/tmp/rpmscanner.* && exec clair
```

Comment:
The `rm`s shouldn't be needed, now.

Is there some way to double-check that?

Reply:
We can remove them and check the PVC metrics once it's deployed